packages/runtime/src/widgets/Search.tsx

```diff
@@ -81,7 +80,6 @@ export const Search: React.FC<SearchProps> = ({
             ? getColor(styles.borderColor)
             : undefined,
           boxShadow: "none",
-          backgroundColor: styles?.backgroundColor,
```

Comment:
Why remove this instead of specifying color in the YAML?

Reply:
actually this applies color only to search input and not to whole search bar. Which is not desired. imo it should applied to whole search bar and not only specific part of it

Reply:
One option is to use the widget ID from the registered binding and assign it to the root element. Then you can use the id in your CSS selector since it's generated from JS